hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/HiveSyncContext.java

```diff
@@ -119,6 +120,7 @@ public static Properties buildSyncConfig(Configuration conf) {
     props.setPropertyIfNonNull(HIVE_SKIP_RO_SUFFIX_FOR_READ_OPTIMIZED_TABLE.key(), String.valueOf(conf.getBoolean(FlinkOptions.HIVE_SYNC_SKIP_RO_SUFFIX)));
     props.setPropertyIfNonNull(META_SYNC_ASSUME_DATE_PARTITION.key(), String.valueOf(conf.getBoolean(FlinkOptions.HIVE_SYNC_ASSUME_DATE_PARTITION)));
     props.setPropertyIfNonNull(HIVE_SYNC_TABLE_STRATEGY.key(), String.valueOf(conf.getString(FlinkOptions.HIVE_SYNC_TABLE_STRATEGY)));
+    props.setPropertyIfNonNull(HIVE_CREATE_MANAGED_TABLE.key(), String.valueOf(conf.getBoolean(FlinkOptions.HIVE_SYNC_CREATE_MANAGED_TABLE)));
     return props;
```

Comment:
No need to add the option because in line 99, we already include all the hudi options in flink configuration.